apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -67,7 +67,7 @@ export function validatorsTableData(
                     commission: Number(validator.commissionRate) / 100,
                     img: img,
                     address: validator.suiAddress,
-                    lastReward: Number(event?.pool_staking_reward) || 0,
+                    lastReward: Number(event?.pool_staking_reward),
```

Comment:
What happens if the `pool_staking_reward` is null or undefined ?

Reply:
Just updated for number check